### DISCUSSION TITLE
Confused about recently added distil-whisper support (CPU?)

### DISCUSSION THREAD

**Discussion by Arche151** [SYSTRAN/faster-whisper#713] [Q&A]
First of all, I wanna say how thankful I am for faster-whisper and that I'm using it every single day! 

I saw, that two days ago, distil-whisper support was added to faster-whisper. My question: Can I use distil-whisper inside faster-whisper via CPU? 

Distil-whisper itself supports CPU, but in the faster-whisper docs only GPU transcription with distil-whisper is mentioned.

And lastly, if I can use distil-whisper inside faster-whisper via CPU, how does its performance compare to normal faster-whisper CPU transcription?

Thanks in advance!

**Comment by ddorian** [comment]
> Can I use distil-whisper inside faster-whisper via CPU?

Yes.

> And lastly, if I can use distil-whisper inside faster-whisper via CPU, how does its performance compare to normal faster-whisper CPU transcription?

It should be faster. Benchmark it.
